modules/core/04-channel/types/timeout_test.go

```diff
@@ -0,0 +1,109 @@
+package types_test
+
+import (
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+)
+
+func (suite *TypesTestSuite) TestTimeoutPassed() {
+	var timeout types.Timeout
+	var passed bool
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"client height is not after timeout height",
+			func() {
+				passed = timeout.AfterHeight(clienttypes.NewHeight(0, 75))
```

Comment:
I think it could be worth adding some more testcases like where heights are equal, where heights have different revision numbers but equal heights etc. 

Similarly for timestamp, where timestamps are equal it should return false